x-pack/plugin/frozen-indices/src/internalClusterTest/java/org/elasticsearch/index/engine/frozen/FrozenIndexIT.java

```diff
@@ -329,9 +334,10 @@ public void testOpenPointInTimeWithNoIndexMatched() {
             );
             final String pitId = client().execute(OpenPointInTimeAction.INSTANCE, openPointInTimeRequest).actionGet().getPointInTimeId();
             try {
-                SearchResponse resp = prepareSearch().setPreference(null).setPointInTime(new PointInTimeBuilder(pitId)).get();
-                assertNoFailures(resp);
-                assertHitCount(resp, 0);
+                assertNoFailuresAndResponse(
+                    prepareSearch().setPreference(null).setPointInTime(new PointInTimeBuilder(pitId)),
+                    searchResponse -> assertHitCount(searchResponse, 0)
+                );
```

Comment:
`assertHitCount` only checking hits count. If we need to double check for no failures, we should have a `asserNoFailuresAndHitCount`